### PR TITLE
Encryption and Support Bundle Changes.

### PR DESCRIPTION
**Pull request by prathameshrodi**
# Backend

## Problem Statement
<pre>
  <code>
  Story Ref (if any): 
Changes for Support Bundle to read rc rather than err messages.
  </code>
</pre>
## Unit testing on RPM done
<pre>
  <code>
  Yes
  </code>
</pre>
## Problem Description
<pre>
  <code>
Need to remove the changes for Support Bundle and  remove decryption from cortxcli
</code>
</pre>
## Solution
<pre>
  <code>
Made the changes as required.
  </code>
</pre>
## Unit Test Cases
<pre>
  <code>
    Locally Tested
  </code>
</pre>

### CODE REVIEW

File: csm/cli/support_bundle/bundle_generate.py [hunk 1]
```diff
@@ -78,7 +78,7 @@ def exc_components_cmd(commands: List, bundle_id: str, path: str,
             cmd_proc = SimpleProcess(f"{command} {bundle_id} {path}")
             output, err, return_code = cmd_proc.run()
             Log.debug(f"Command Output -> {output} {err}, {return_code}")
-            if err:
+            if return_code == -1:
```
Comment (navalkp) [inline csm/cli/support_bundle/bundle_generate.py:81]: Is -1 only failed case. mostly other than 0 is failure

Comment (ajaysrivas) [reply csm/cli/support_bundle/bundle_generate.py:81]: It should be return_code != 0:

Comment (prathameshrodi) [reply csm/cli/support_bundle/bundle_generate.py:81]: will change

Comment (prathameshrodi) [reply csm/cli/support_bundle/bundle_generate.py]: ChangedFixed

**Pull request closed without merging**